stacks/Api.ts

```diff
@@ -189,7 +181,7 @@ export function ApiStack({ stack }: StackContext) {
         environment: {
             STREET_MANAGER_SQS_QUEUE_URL: streetManagerSqsQueue.queueUrl,
             TEST_STREET_MANAGER_TOPIC_ARN: streetManagerTestTopic?.topicArn ?? "",
-            STREET_MANAGER_MESSAGE_TOPIC_ARN: streetManagerMessageTopic?.topicArn ?? "",
+            STREET_MANAGER_MESSAGE_TOPIC_ARN: streetManagerTestTopic?.topicArn ?? "",
```

Comment:
We can remove this environment variable as we already have `TEST_STREET_MANAGER_TOPIC_ARN`